backend/danswer/connectors/file/connector.py

```diff
@@ -0,0 +1,124 @@
+import os
+import zipfile
+from collections.abc import Generator
+from enum import Enum
+from pathlib import Path
+from typing import Any
+from typing import IO
+
+from danswer.configs.app_configs import INDEX_BATCH_SIZE
+from danswer.configs.constants import DocumentSource
+from danswer.connectors.file.utils import check_file_ext_is_valid
+from danswer.connectors.file.utils import get_file_ext
+from danswer.connectors.interfaces import GenerateDocumentsOutput
+from danswer.connectors.interfaces import LoadConnector
+from danswer.connectors.models import Document
+from danswer.connectors.models import Section
+from danswer.utils.logging import setup_logger
+
+
+logger = setup_logger()
+
+_LINK_METADATA_FLAG = "#LINK="
```

Comment:
I prefer explicitly mentioning danswer in it so there's even less chance of accidental hit or whatever. 
#DANSWER_METADATA={"link":"whatever"}

how do we feel about this style?

Reply:
sure, seems good